apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -249,13 +244,20 @@ class MoveStudents extends Component {
     });
   };
 
+  isMovableSection = (section) => {
+    if (section.id === this.props.currentSectionId) { return false; }
+    if (section.loginType === SectionLoginType.google_classroom || section.loginType === SectionLoginType.clever) { return false; }
+    return true;
```

Comment:
Three thoughts:

1. I think we generally avoid one-line/inline blocks, (`if (x) { return true; }`) but I can't find a spot in our styleguide that talks about this and we're obviously not enforcing it via linter so I guess they're fine.  On the other hand, line 249 is pretty long.

2. I'd reverse your `SectionLoginType` condition to include word/picture/email instead of excluding google_classroom/clever, because it won't need to change as we add new OAuth+rostered section types.

3. I'm fine with early returns but I wonder if this would self-document better by extracting some named variables, e.g.

Reply:
i like all of those points! i was debating which way the logic should go (whether to exclude google/clever or to explicitly include the others) and i think you make a good point. the same logic applies in dashboard (the endpoint is currently enforcing that the `login_type` of the section isn't google or clever), so i'll invert the logic there, too

Reply:
re (1), I couldn't find this in our style guide either, but we do try to be consistent about this. The good news is that we could easily start enforcing this with a lint rule which does not have too many existing violations. I'll send a follow-up PR.